platforms/Linux/DEB/Ubuntu/shared/debian/rules

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/make -f
```

Comment:
This file did not get recognised as a moved file, so I am listing the changes I made systematically below.